engine/schema/src/com/cloud/upgrade/DatabaseUpgradeChecker.java

```diff
@@ -242,13 +243,13 @@ public DatabaseUpgradeChecker() {
 
         _upgradeMap.put("4.5.2", new DbUpgrade[] {new Upgrade452to460(), new Upgrade460to461(), new Upgrade461to470()});
 
-        _upgradeMap.put("4.5.3", new DbUpgrade[] {new Upgrade453to460()});
+        _upgradeMap.put("4.5.3", new DbUpgrade[] {new Upgrade453to460(), new Upgrade460to461(), new Upgrade461to470()});
 
         _upgradeMap.put("4.6.0", new DbUpgrade[] {new Upgrade460to461(), new Upgrade461to470()});
 
         _upgradeMap.put("4.6.1", new DbUpgrade[] {new Upgrade461to470()});
 
-        _upgradeMap.put("4.6.2", new DbUpgrade[] {new Upgrade461to470()});
+        _upgradeMap.put("4.6.2", new DbUpgrade[] {new Upgrade462to470()});
```

Comment:
I've explained the reason in detail on dev@. The upgrade path used for 4.6.2 version (source) is Upgrade461to470 that assumes from version 4.6.1 to 4.7.0 in getUpgradableVersionRange(). The fix tries to be consistent with the semantics followed in writing/using upgrade paths. Till upgrade process changes and new tools are introduced, I would like to be consistent with the pattern we've here.